teerex/sgx-verify/fuzz/fuzz_targets/signature_check.rs

```diff
@@ -0,0 +1,29 @@
+#![no_main]
+#![feature(core_panic)]
+
+pub extern crate alloc;
```

Comment:
Very happy to see some fuzzing!

Reply:
It is a first step, but to provide real value we would need CI integration. I added a comment regarding what is missing in #99.

Reply:
I agree. Still, thanks for the start!